projects/views.py

```diff
@@ -97,9 +99,9 @@ def update(self, request, pk):
         try:
             if not pk:
```

Comment:
Lines 100 and 101 seem redundant

Reply:
Removed that. I agree because we cannot put non integer value, so we don't need that.